RayLin.md

```diff
@@ -1,22 +1,22 @@
-# 07/04 ~ 07/08
+# 07/11 ~ 07/15
 
-- [Firefox for Android]
-  - Bug 1086911 - Update style of about:firefox
-    - wait :antlam reply, will NI again this week
-  - Bug 1079504 - Improve about:addons if user has no add-ons installed
-    - feedback-
-    - NI pwalmsley
-  - Bug 1027535 - Addon Manager doesn't provide enough details in case of blocklisted addons
-    - provided screenshot but blocked by add-ons signing meta bug: Bug 1170113
-  - Bug 1282678 - [webvtt] Consider adding closed caption button to mobile video control
-    - redirected to Barbara to prioritize
-- [Firefox]
+- [Video Control]
   - Bug 1281414 - [webvtt] Update the design of closed caption button icon
-    - NI :shorlander
-  - Bug 1198279 - Closing Find Toolbar should clear all highlighted matches
-    - FIXED
+    - CC'd Carol, attcah comparison screenshot and reiterate intention
+  - Bug 883122 - [webvtt] Implement the rules for constructing a chapter tree from a text track
+    - build test page and study algorithm
+  - Bug 1281414 - [webvtt] Update the design of closed caption button icon
+    - discuss with Peko
+      - refresh primitive features without regard to Bug 883122
+  - Bug 681553 - [meta] Make HTML5 <audio> controls less of a hack
+    - NI :Dolske
+    - ask if it doable and what's the expected result
   - Bug 855149 - Make <input type=range> usable for media controls
-    - Made an input range prototype and asked for feedback
-    - A couple concerns:
-      - Duplicate works of refactoring to web component and de-xul.
-      - JS workaround comes with its complications
+    - need to figure out event issues:
+      - how to avoid event propagating to outside
+      - how to implement system group event option without XBL
+
+- [Firefox for Android]
+  - Bug 1282678 - [webvtt] Consider adding closed caption button to mobile video control
+    - :barbars is ok with it, and added :antlam for feedback
```

Comment:
Nice. Thanks for spending time doing the engagement while keeping me in the loop.